docs/RegisteredFilterPlugins.md

```diff
@@ -568,201 +565,201 @@ Most stages support samples of 8 to 24 bits (big/little endian, signed/unsigned)
 
 The FAPEC library and CLI runs on Linux, Windows and Mac. The HDF5 user must request and install the library separately, thus allowing to upgrade it without requiring changes in your HDF5 code.
 
-#### Filter Information:
+#### Information
 
 https://www.dapcom.es/fapec/
 https://www.dapcom.es/get-fapec/
 https://www.dapcom.es/resources/FAPEC_EndUserLicenseAgreement.pdf
 
-#### Contact Information:
+#### Contact
 
 Jordi Portell i de Mora (DAPCOM Data Services S.L.)
 
 fapec at dapcom dot es
 
-##
+---
 
-<h2 id="bitgroom">BitGroom</h2>
+### BitGroom <a name="bitgroom"></a>
 
-#### Filter ID:  32022
+Filter ID:  32022
 
-#### Filter Description:
+#### Description
 
 The BitGroom quantization algorithm is documented in:
 
 Zender, C. S. (2016), Bit Grooming: Statistically accurate precision-preserving quantization with compression, evaluated in the netCDF Operators (NCO, v4.4.8+), Geosci. Model Dev., 9, 3199-3211, doi:10.5194/gmd-9-3199-2016.
 
-#### Filter Information:
+#### Information
 
 The filter is documented and maintained in the Community Codec Repository (https://github.com/ccr/ccr).
 
-#### Contact Information:
+#### Contact
 
 Charlie Zender  (University of California, Irvine)
 
-##
+---
 
-<h2 id="gbr">Granular BitRound (GBR)</h2>
+### Granular BitRound (GBR) <a name="gbr"></a>
 
-#### Filter ID:  32023
+Filter ID:  32023
 
-#### Filter Description:
+#### Description
 
 The GBG quantization algorithm is a significant improvement the BitGroom filter documented in:
 
 Zender, C. S. (2016), Bit Grooming: Statistically accurate precision-preserving quantization with compression, evaluated in the netCDF Operators (NCO, v4.4.8+), Geosci. Model Dev., 9, 3199-3211, doi:10.5194/gmd-9-3199-2016.
 
-#### Filter Information:
+#### Information
 
 This filter is documented, implemented, and maintained in the Community Codec Repository (https://github.com/ccr/ccr).
 
-#### Contact Information:
+#### Contact
 
 Charlie Zender  (University of California, Irvine)
 
-##
+---
 
-<h2 id="sz3">SZ3</h2>
+### SZ3 <a name="sz3"></a>
 
-#### Filter ID:  32024
+Filter ID:  32024
 
-#### Filter Description:
+#### Description
 
 SZ3 is a modular error-bounded lossy compression framework for scientific datasets, which allows users to customize their own compression pipeline to adapt to diverse datasets and user-requirements. Compared with SZ2 (filter id: 32017), SZ3 has integrated a more effective prediction such that its compression qualities/ratios are much higher than that of SZ2 in most of cases.
```

Comment:
extra work 'of' in the end of the paragraph 'in most of cases'.

Reply:
Fixed.